lexicons/tools/ozone/server/getConfig.json

```
@@ -0,0 +1,60 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.server.getConfig",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get details about ozone's server configuration.",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "appview": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "pds": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "blobDivert": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "chat": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "viewer": {
+              "type": "ref",
+              "ref": "#viewerConfig"
+            }
+          }
+        }
+      }
+    },
+    "serviceConfig": {
+      "type": "object",
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        }
+      }
+    },
+    "viewerConfig": {
+      "type": "object",
+      "properties": {
+        "role": {
+          "type": "string",
+          "knownValues": [
+            "tools.ozone.moderator.defs#modRoleAdmin",
+            "tools.ozone.moderator.defs#modRoleModerator",
+            "tools.ozone.moderator.defs#modRoleTriage"
```

Syncing up with the changes from https://github.com/bluesky-social/atproto/pull/2460.  As soon as this is landed and codegen'ed, we can launch this!
```suggestion
            "tools.ozone.team.defs#roleAdmin",
            "tools.ozone.team.defs#roleModerator",
            "tools.ozone.team.defs#roleTriage"
```